### PR TITLE
[hi] Use localized text for Hindi language configuration

### PR DESCRIPTION
**Pull request by sftim**
```diff
- description = "Production-Grade Container Orchestration"
+ description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
```
and
```diff
- languageName = "Hindi"
+ languageName = "हिन्दी"
```
/language hi

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [hi] Use localized text for Hindi language configuration

File: config.toml [hunk 1]
```diff
@@ -427,8 +427,8 @@ language_alternatives = ["en"]
 
 [languages.hi]
 title = "Kubernetes"
-description = "Production-Grade Container Orchestration"
-languageName = "Hindi"
+description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
+languageName = "िनक्सa"
```
Comment (championshuttler) [inline config.toml:431]:
```suggestion
languageName = "हिन्दी"
```

**Review by championshuttler — COMMENTED** [review]
Thanks

File: config.toml (hunk 1 above)
Comment (sftim) [reply config.toml:431]: Oops.

**Review by championshuttler — APPROVED** [review]
Looks good to me, thanks

**Comment by anubha-v-ardhan** [comment]
/lgtm
Thanks @sftim

**Review by divya-mohan0209 — APPROVED** [review]
lgtm and therefore approving.

**Comment by sftim** [comment]
@divya-mohan0209 is a SIG Docs chair - but Prow doesn't know that's the case for this branch.

/approve

**Pull request merged**